.github/workflows/bitcoin-tests.yml

```diff
@@ -1,205 +1,173 @@
+          - tests::neon_integrations::fuzzed_median_fee_rate_estimation_test_window10
+          - tests::neon_integrations::fuzzed_median_fee_rate_estimation_test_window5
           - tests::neon_integrations::liquid_ustx_integration
-          - tests::neon_integrations::stx_transfer_btc_integration_test
-          - tests::neon_integrations::stx_delegate_btc_integration_test
-          - tests::neon_integrations::bitcoind_forking_test
-          - tests::neon_integrations::should_fix_2771
-          - tests::neon_integrations::pox_integration_test
+          - tests::neon_integrations::microblock_fork_poison_integration_test
+          - tests::neon_integrations::microblock_integration_test
+          - tests::neon_integrations::microblock_large_tx_integration_test_FLAKY
+          - tests::neon_integrations::microblock_limit_hit_integration_test
+          - tests::neon_integrations::miner_submit_twice
           - tests::neon_integrations::mining_events_integration_test
-          - tests::bitcoin_regtest::bitcoind_integration_test
-          - tests::should_succeed_handling_malformed_and_valid_txs
+          - tests::neon_integrations::pox_integration_test
+          - tests::neon_integrations::push_boot_receipts
+          - tests::neon_integrations::runtime_overflow_unconfirmed_microblocks_integration_test
+          - tests::neon_integrations::should_fix_2771
+          - tests::neon_integrations::size_check_integration_test
+          - tests::neon_integrations::size_overflow_unconfirmed_invalid_stream_microblocks_integration_test
           - tests::neon_integrations::size_overflow_unconfirmed_microblocks_integration_test
           - tests::neon_integrations::size_overflow_unconfirmed_stream_microblocks_integration_test
-          - tests::neon_integrations::size_overflow_unconfirmed_invalid_stream_microblocks_integration_test
-          - tests::neon_integrations::runtime_overflow_unconfirmed_microblocks_integration_test
-          - tests::neon_integrations::antientropy_integration_test
-          - tests::neon_integrations::filter_low_fee_tx_integration_test
-          - tests::neon_integrations::filter_long_runtime_tx_integration_test
-          - tests::neon_integrations::microblock_large_tx_integration_test_FLAKY
-          - tests::neon_integrations::block_large_tx_integration_test
-          - tests::neon_integrations::microblock_limit_hit_integration_test
-          - tests::neon_integrations::block_limit_hit_integration_test
-          - tests::neon_integrations::fuzzed_median_fee_rate_estimation_test_window5
-          - tests::neon_integrations::fuzzed_median_fee_rate_estimation_test_window10
-          - tests::neon_integrations::use_latest_tip_integration_test
-          - tests::neon_integrations::test_flash_block_skip_tenure
+          - tests::neon_integrations::stx_delegate_btc_integration_test
+          - tests::neon_integrations::stx_transfer_btc_integration_test
```

Comment:
Is there any way to autogenerate this so we don't need to update it manually?

Reply:
sort of...i tested using nextest filters https://nexte.st/book/filter-expressions.html

ex (from my notes, have to adjust to run the tests):
```
cargo nextest list \
   --run-ignored all \
   --archive-file /tmp/test_archive.tar.zst \
   -E '(
      test(/^tests::neon_integrations::*/) &
      test(/^tests::bitcoin_regtest::*/) &
      not test(/^tests::neon_integrations::atlas*/)
   )'
```

and the resultant list of tests matched what i expected the runner to run. 
however, doing it this way causes the tests to run serially, which increases the runtime of the test(s). 


using a matrix with a nextest partition (https://nexte.st/book/partitioning.html) is an option, but then we're getting into a scenario where we have to decide how many concurrent tests to run. 8? 12? 32? etc.

an argument can be made to use these filters though, the result will be slower tests and more time running the CI. as well as a little more care needed in deciding how many separate jobs to run - i found it useful to know exactly how many tests were being run, and seeing what test had failed in the UI vs a generic (partition 1 failed) and having to consult the logs to see what test(s) had failed. 

i do not have a strong opinion here - i can see the benefit of both approaches and will agree with the consensus, whatever it may be.

Reply:
you can see an example of this partitioning here: https://github.com/stacks-network/stacks-blockchain/blob/feat/faster-ci/.github/workflows/stacks-blockchain-tests.yml#L140

i felt using a filter as well as a partition was a bit too messy - but it's totally doable if others feel strongly about going this way.